pvlib/iotools/solargis.py

```diff
@@ -0,0 +1,171 @@
+"""Functions to retrieve and parse irradiance data from Solargis."""
+
+import pandas as pd
+import requests
+import io
+
+URL = 'https://solargis.info/ws/rest/datadelivery/request?'
+
+VARIABLE_MAP = {
+    'GHI': 'ghi',
+    'GHI_C': 'ghi_clear',  # this is stated in documentation
+    'GHIc': 'ghi_clear',  # this is used in practice
+    'DNI': 'dni',
+    'DNI_C': 'dni_clear',
+    'DNIc': 'dni_clear',
+    'DIF': 'dhi',
+    'GTI': 'poa_global',
+    'GTI_C': 'poa_global_clear',
+    'GTIc': 'poa_global_clear',
+    'SE': 'solar_elevation',
+    'SA': 'solar_azimuth',
+    'TEMP': 'temp_air',
+    'TD': 'temp_dew',
+    'AP': 'pressure',
+    'RH': 'relative_humidity',
+    'WS': 'wind_speed',
+    'WD': 'wind_direction',
+    'INC': 'aoi',  # angle of incidence of direct irradiance
+    'PWAT': 'precipitable_water',  # [kg/m2]
```

Comment:
At least some functions in pvlib expect units of cm for pwat.  Worth looking through pvlib in more detail and seeing if we should convert units here

Reply:
The following functions use or calculate precipitable_water in units of cm: kasten96_lt, gueymard94_pw, spectral_factor_firstsolar, spectral_factor_caballero, simplified_solis, bird.

Notably, the Solcast iotools convert it from kg/m2 to cm. The unit from PSM3 is already in cm. In TMY files it's also in cm. Though in TMY2 it seems like its in mm, but in TMY3 it is in cm!

I have added a conversion similar to what is done in the solcast functions.